build.gradle

```diff
@@ -34,11 +34,12 @@ description = 'New code for Egeria'
 
 // TODO: Specify Dependency versions as variables
 ext {
-    egeriaversion = '3.6'
+    egeriaversion = '3.10'
     slf4jVersion = '1.7.36'
     jacksonVersion = '2.13.2'
     jupiterVersion = '5.8.2'
     httpclientVersion = '4.5.13'
+    mockitoVersion ='4.5.1'
```

Comment:
I think we need a settings.gradle file to supply the root project name. Unless there is some other way to do this.